node/src/components/contract_runtime.rs

```diff
@@ -563,10 +584,15 @@ impl ContractRuntime {
         finalized_block: FinalizedBlock,
         deploys: Vec<Deploy>,
         transfers: Vec<Deploy>,
+        activation_point: ActivationPoint,
+        activation_point_era_id: Option<u64>,
```

Comment:
```suggestion
        activation_point_block_height: Option<u64>,
```

Reply:
I renamed it to `key_block_height_for_activation_point` - it's not the height of the activation point, but the height of an era previous to the activation point (which I think in other places is referred to as a "key block")